src/Steppenwolf/Pages/_Host.cshtml

```diff
@@ -8,28 +8,33 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>Steppenwolf</title>
-    <base href="~/" />
-    <link href="/css/style.css" rel="stylesheet" />
+    <base href="~/"/>
+    <!-- TODO: this should be css -->
+    <link href="/scss/style.scss" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```

Comment:
Don't we have bootstrap already included in the style.scss?

Reply:
We have half of what we need. I've added the cdns as 'quick' fix and raised https://github.com/043Tech/Steppenwolf/issues/24 for proper fix.